include/init-css/init-css.h

```diff
@@ -0,0 +1,13 @@
+#ifndef INIT_CSS_H
+#define INIT_CSS_H
+
+#include <stdio.h>
+#include <stdlib.h>
+
+/**
+ * Creates CSS file

```

Comment:
This function does not create CSS file. It creates and adds styles in CSS structure.